leveldbcache/leveldbcache.go

```diff
@@ -0,0 +1,51 @@
+// Package leveldbcache provides an implementation of httpcache.Cache that
+// uses github.com/syndtr/goleveldb/leveldb
+package leveldbcache
+
+import (
+	"github.com/syndtr/goleveldb/leveldb"
+)
+
+// Cache is an implementation of httpcache.Cache with leveldb storage
+type Cache struct {
+	db *leveldb.DB
+}
+
+// Get returns the response corresponding to key if present
+func (c *Cache) Get(key string) (resp []byte, ok bool) {
+	var err error
+	resp, err = c.db.Get([]byte(key), nil)
+	if err != nil {
+		return []byte{}, false
```

Comment:
You could write `return nil, false` here. A `nil` slice has the same properties as a zero-element slice (namely, its `len` is 0, and it can be appended to). It's a more common practice.